packages/sdk/src/calcPreVerificationGas.ts

```diff
@@ -51,17 +51,18 @@ export const DefaultGasOverheads: GasOverheads = {
  * it is based on parameters that are defined by the Ethereum protocol for external transactions.
  * @param userOp filled userOp to calculate. The only possible missing fields can be the signature and preVerificationGas itself
  * @param overheads gas overheads to use, to override the default values
+ * @param sigSize the expected length of the userOp signature, used to generate a dummy value.
  */
-export function calcPreVerificationGas (userOp: Partial<NotPromise<UserOperationStruct>>, overheads?: Partial<GasOverheads>): number {
+export function calcPreVerificationGas (userOp: Partial<NotPromise<UserOperationStruct>>, overheads?: Partial<GasOverheads>, sigSize?: number): number {
```

Comment:
Instead if adding an extra parameter (and I'm sure there are more..), it is better to add it to the struct.
The usage becomes
calcPreVerificationGas(userOp, { sigSize:65 })